src/main/java/uk/gov/hmcts/reform/ccd/test/stubs/service/token/JWTokenParser.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.ccd.test.stubs.service.token;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import java.security.Key;
+
+public final class JWTokenParser {
+
+    private JWTokenParser() {
+    }
+
+    public static Claims parseToken(String publicKeyFileName, String jwt) {
+        final Key signingKey = KeyUtil.getPublicKey(publicKeyFileName);
```

Comment:
where do we use this ?

Reply:
we are not using this... but will be useful for testing...